### DISCUSSION TITLE
There are so many models available in coqui, Can someone point me the best model out of them, which gave the best results so far.

### DISCUSSION THREAD

**Discussion by Rajashekhar-Reddy** [coqui-ai/TTS#2973] [General Q&A]
I'm too confused about choosing the perfect modal.
Appreciate it, if anyone could add the output results here for the suggested modal.

**Comment by imjonos** [comment]
Same question)

**Comment by thorstenMueller** [comment]
Hi @Rajashekhar-Reddy and @imjonos. "Best" and "perfect" models are very subjective and individual, but maybe you can get an idea by listening to them in my audio sample comparison videos.
* https://youtu.be/HojuVmW5LUI
* https://youtu.be/Vnjv2L31eyQ

**Reply by imjonos** [reply]
Thanks 👍

**Reply by pivolan** [reply]
if you want train new models, at the current time the best results for me when training from the scratch on small datasets 3-5 hours, best results I got from vits model single voice. Also if you want finetune your own voice after opensource model, there are also two experiments was on my side:
1. finetune single voice - got much more clear audio voice, but create errors in prnounciations, minimum dataset for finetune 1hours, good results will be from 4 hours, without pronounce errors.
2. finetune using yourtts with speaker encoder d_vector, usefull even you have only 15 minutes, and finetune it will be responsible, and good enough quality. If you have 1 hours voice not so clear as in single voice, but much less errors in prnounciation. And for 4 hours I didn't find difference in pronounce errors, but in single voice model audio much more clear.

**Comment by waqasawan9996** [comment]
which model contain that polly voices specifically matthew

**Reply by eginhard** [reply]
Polly is Amazon's commercial TTS offering: https://docs.aws.amazon.com/polly/latest/dg/API_Voice.html

**Reply by waqasawan9996** [reply]
hmm i was looking for a personal use that should sounds like matthew  is there any pretrained models for coqui that should good in storytellig
